src/tools/ScenarioMeasurement/Util/ProcessHelper.cs

```diff
@@ -38,7 +38,7 @@ public class ProcessHelper
         /// <param name="appArgs">Optional arguments</param>
         /// <param name="workingDirectory">Optional working directory (defaults to current directory)</param>
         /// <returns></returns>
-        public Result Run()
+        public (Result, int) Run()
```

Comment:
Further thought from our conversation. This is fine as it is. One minor advantage of moving the names here (instead of tuple deconstruction) is that it gives you an idea of what the intent of these fields is when you are looking at the method. `(Result result, int pid)` tells me immediately what is going on. Still up to you but I think it's a minor point in favor of a change.